pkg/utils/parse_arf_result_test.go

```diff
@@ -0,0 +1,37 @@
+package utils
+
+import (
+	complianceoperatorv1alpha1 "github.com/openshift/compliance-operator/pkg/apis/complianceoperator/v1alpha1"
+	"io/ioutil"
+	"testing"
+)
+
+func TestParseRemediationsFromArf(t *testing.T) {
+	expName := "testScan-no-empty-passwords"
+
+	arf, err := ioutil.ReadFile("results.arf.xml")
+	if err != nil {
+		t.Error(err)
+	}
+
+	remList, err := ParseRemediationsFromArf("testScan", "testNamespace", string(arf))
+	if err != nil {
+		t.Error(err)
+	}
+
+	if len(remList) != 1 {
+		t.Errorf("Expected one remediation, got %d", len(remList))
+	}
+
+	// FIXME: maybe define Equal methods on the type and use go-cmp/cmp ?
```

Comment:
that would be a nice cleanup. It could be done in a further iteration.

Reply:
OK, added to my todo list

Reply:
in another PR. No need to do it in this one.